app/templates/main/map.html

```diff
@@ -0,0 +1,61 @@
+{% extends 'layouts/base.html' %}
+{% block content %}
+<html>
+    <head>
+        <link rel="stylesheet" href="static/styles/iThing.css" type="text/css" />
+        <link rel="stylesheet" href="static/styles/map.css" type="text/css" />
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+        <script src="http://maps.googleapis.com/maps/api/js"></script>
+        <script src="static/js/jquery-ui.js"></script>
+        <script src="static/js/jQDateRangeSlider-min.js"></script>
+        <script src="static/js/initialize_map.js"></script>
+        <meta id="vehicle_ids" data-name="{{vehicle_ids}}">
+        <meta id="license_plates" data-name="{{license_plates}}">
+        <meta id="latitudes" data-name="{{latitudes}}">
+        <meta id="longitudes" data-name="{{longitudes}}">
+        <meta id="dates" data-name="{{dates}}">
+        <meta id="durations" data-name="{{durations}}">
+        <meta id="agencies" data-name="{{agencies}}">
+        <meta id="pictures" data-name="{{pictures}}">
+        <meta id="descriptions" data-name="{{descriptions}}">
+    </head>
+    <body>
+     <div class ="column">
+         <div id="wrapper">
+             <div class="row">
+                <div id="googleMap"></div>
+                           <div class="ui form">
+              <div class="fields">
+                <div class="three wide field">
+                  <label>License Plate Number</label>
+                  <input type="text" placeholder="XXX-XXX">
+                </div>
+                <div class="three wide field">
+                  <label>Vehicle ID</label>
+                  <input type="text" placeholder="enter vehicle ID">
+                </div>
+              </div>
+              <div class="fields">
+                <div class="six wide field">
+                    <label>Notes </label>
+                  <input type="text" placeholder="(e.g. 'Was the driver sleeping?')">
+                </div>
+                <div class="three wide field">
+                    <label>Idling Duration </label>
+                  <input type="number" placeholder="~X minutes">
+                </div>
+            </div>
+
+
+             </div>
+    	    <div id="slider"></div>
+            <form name = "addressForm" id = "addressForm">
```

Comment:
wrap it around a semantic ui search http://semantic-ui.com/modules/search.html or try adding the class to the form and see if that changes it too